mozilla/fx-private-relay
# Show numpad when entering a verification code

# New feature description

Verification codes, as far as I know, only consist of numbers, so mobile phones can show a numpad there.

# Screenshot (if applicable)

![Screenshot_20230109-121406_Firefox_Nightly](https://user-images.githubusercontent.com/4251/211295855-8fe5ebed-59f9-413f-966b-8ec459ddf00f.png)

# How to test

Open the phone onboarding on mobile, and enter your phone number. When you get to the verification step and focus the input field, your phone's keyboard should be in numpad mode.

# Checklist (Definition of Done)
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege. N/A
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process. N/A
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage. N/A
